readthedocs/settings/base.py

```diff
@@ -116,6 +116,7 @@ def INSTALLED_APPS(self):  # noqa
         if donate:
             apps.append('django_countries')
             apps.append('readthedocsext.donate')
+            apps.append('readthedocsext.search')
```

Comment:
Why is this under `if donate:`?

Reply:
I fixed that here, actually: https://github.com/rtfd/readthedocs.org/pull/3356/files#diff-5973fe59a4d52278d55f884d145bd5d7R11 :) Waiting on it to get merged.

Reply:
Merged #3356, this can be rebased